### PR TITLE
fixed #99

### LINKED ISSUE #99: BAKE button should disappear from Pie pages when not available
On the individual Pie page we are showing a Bake button but it should be visible only if an oven for that specific Pie is available

### PR DESCRIPTION
**Pull request by TotoFullStackDev**
##### Checklist

- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Test coverage hasn't decreased

### CODE REVIEW

**Comment by Alexintosh** [comment]
@Chiptuttofuso Checking the pie by symbol is not advisable, instead cross-reference the pie address with the Oven config file to determine if an oven is available.

Oven config file: https://github.com/pie-dao/pools.piedao.org/blob/development/src/config/ovensConf.js

**Comment by TotoFullStackDev** [comment]
> @Chiptuttofuso Checking the pie by symbol is not advisable, instead cross-reference the pie address with the Oven config file to determine if an oven is available.
> 
> Oven config file: https://github.com/pie-dao/pools.piedao.org/blob/development/src/config/ovensConf.js

@Alexintosh done, this can be closed now.

File: src/pages/PieIndexLanding.svelte [hunk 1]
```diff
@@ -45,6 +46,17 @@
   }
 
   $: token = params.address;
+  const isBakingPie = (address) => {
```
Comment (Alexintosh) [inline src/pages/PieIndexLanding.svelte:47]: this is your friend https://lodash.com/docs/4.17.15#find

File: src/pages/ExperiPie.svelte [hunk 2]
```diff
@@ -51,6 +52,18 @@
 
   $: token = params.address;
 
+  const isBakingPie = (address) => {
```
Comment (Alexintosh) [inline src/pages/ExperiPie.svelte:55]: https://lodash.com/docs/4.17.15#find

**Review by Alexintosh — CHANGES_REQUESTED** [review]

File: src/pages/PieIndexLanding.svelte (hunk 1 above)
Comment (TotoFullStackDev) [reply src/pages/PieIndexLanding.svelte:47]: @Alexintosh done.

File: src/pages/ExperiPie.svelte (hunk 2 above)
Comment (TotoFullStackDev) [reply src/pages/ExperiPie.svelte:55]: @Alexintosh done.

**Review by Alexintosh — APPROVED** [review]

**Pull request merged**